meson.build

```diff
@@ -262,7 +262,7 @@ if get_option('enable-tflite-interpreter')
   extra_defines += '-DENABLE_TFLITE_INTERPRETER=1'
 endif
 
-gmock_dep = dependency('gmock', static: true, required: false)
+gmock_dep = dependency('gmock', version : '>=1.10.0', static: true, required: false)
```

Comment:
How about if gmock_dep is false, then skip the unittest for profile in line 54 at test/unittest/meson.build
``` bash
  if get_option('enable-profile') and gmock_dep
     test_target += [['unittest_nntrainer_profiler', []]] 
  endif
```

Reply:
thanks. it changed to skip only profile test.